apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V27__Create_BeneficiaryMonthlytable.sql

```diff
@@ -0,0 +1,141 @@
+/*
+ * The pipeline server will populate the table as new beneficaries are added or existing beneficaries are updated. 
+ */
+
+${logic.tablespaces-escape} SET default_tablespace = fhirdb_ts2;
+
+create table "BeneficiaryMonthly" (
+    "parentBeneficiary" varchar(255) not null,
+    "yearMonth" date not null,
+    "fipsStateCntyCode" varchar(5),
+    "medicareStatusCode" varchar(2),
+    "entitlementBuyInInd" char(1),
+    "hmoIndicatorInd" char(1),
+    "partCContractNumberId" varchar(5),
+    "partCPbpNumberId" varchar(3),
+    "partCPlanTypeCode" varchar(3),
+    "partDContractNumberId" varchar(5),
+    "partDPbpNumberId" varchar(3),
+    "partDSegmentNumberId" varchar(3),
+    "partDRetireeDrugSubsidyInd" char(1),
+    "medicaidDualEligibilityCode" varchar(2),
+    "partDLowIncomeCostShareGroupCode" varchar(2),
+    constraint "BeneficiaryMonthly_pkey" primary key ("parentBeneficiary", "yearMonth")
+)
+${logic.tablespaces-escape} tablespace "beneficiarymonthly_ts"
+;
+
+/* we don't know why there are null values for this column, but we'll figure that out later in a different story." */
```

Comment:
Apologies, I'd probably explain a bit more here (folks tend to look at the DB migrations when onboarding, so it's good to explain ourselves in them).
```suggestion
/*
 * Per the CCW folks, the beneEnrollmentReferenceYear column is null for newer bene records that
 * haven't received data from all sources yet. There are over a million such records, which is a bit concerning,
 * but that will be addressed in a separate ticket, if needed.
 */
```

Reply:
changed the comment